src/components/Player.css

```diff
@@ -0,0 +1,16 @@
+.Player {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.Player__score {
+  display: grid;
+  gap: 5px;
+  grid-template-columns: repeat(3, 1fr);
+  place-items: center;
+}
+
+.Player__score > button {
```

Comment:
```suggestion
.Player__button {
```

Reply:
We could also give all buttons in `index.css` a minimal default padding. Or search for a base-css file. I am thinking about a small css library that does what we need. I'll look it up.